<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbManagerDlgTableProperties</class>
 <widget class="QDialog" name="DbManagerDlgTableProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table properties</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabColumns">
      <attribute name="title">
       <string>Columns</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Table columns:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="viewFields">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="btnAddColumn">
           <property name="text">
            <string>Add column</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddGeometryColumn">
           <property name="text">
            <string>Add geometry column</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEditColumn">
           <property name="text">
            <string>Edit column</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>29</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteColumn">
           <property name="text">
            <string>Delete column</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabConstraints">
      <attribute name="title">
       <string>Constraints</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Primary, foreign keys, unique and check constraints:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="viewConstraints">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="btnAddConstraint">
           <property name="text">
            <string>Add primary key / unique</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteConstraint">
           <property name="text">
            <string>Delete constraint</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabIndexes">
      <attribute name="title">
       <string>Indexes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Indexes defined for this table:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="viewIndexes">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_3">
         <item>
          <widget class="QPushButton" name="btnAddIndex">
           <property name="text">
            <string>Add index</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddSpatialIndex">
           <property name="text">
            <string>Add spatial index</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteIndex">
           <property name="text">
            <string>Delete index</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabComment">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Comment</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Comment defined for this table:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="viewComment"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_4">
         <item>
          <widget class="QPushButton" name="btnAddComment">
           <property name="text">
            <string>Add comment</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteComment">
           <property name="text">
            <string>Delete comment</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabs</tabstop>
  <tabstop>viewFields</tabstop>
  <tabstop>btnAddColumn</tabstop>
  <tabstop>btnAddGeometryColumn</tabstop>
  <tabstop>btnEditColumn</tabstop>
  <tabstop>btnDeleteColumn</tabstop>
  <tabstop>viewConstraints</tabstop>
  <tabstop>btnAddConstraint</tabstop>
  <tabstop>btnDeleteConstraint</tabstop>
  <tabstop>viewIndexes</tabstop>
  <tabstop>btnAddIndex</tabstop>
  <tabstop>btnAddSpatialIndex</tabstop>
  <tabstop>btnDeleteIndex</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DbManagerDlgTableProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DbManagerDlgTableProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
